src/System.Runtime.Intrinsics.X86/src/System/Runtime/Intrinsics/X86/Aes.cs

```diff
@@ -0,0 +1,55 @@
+// =+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+
+//
+// Aes.cs
+//
+// A class that implements intrinsic functions to provide access to Intel AES instructions.
+//
+// =+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+
+
+using System;
+using System.Runtime.Intrinsics;
+
+namespace System.Runtime.Intrinsics.X86
+{
+    public static class Aes 
+    {
+        public static bool IsSupported { get; }
```

Comment:
It would be useful to have a comment indicating what the corresponding CPUID check is...